packages/uniforms-material/src/SelectField.js

```diff
@@ -27,40 +26,52 @@ const xor = (item, array) => {
 const renderSelect = ({
   allowedValues,
   disabled,
+  error,
+  errorMessage,
   fieldType,
+  fullWidth,
+  helperText,
   id,
+  InputLabelProps,
```

Comment:
I'd rather keep it compatible... Maybe merge the old and the new one?

Also, ordering.